### DISCUSSION TITLE
Simulating Anymal D robots

### DISCUSSION THREAD

**Discussion by guiskibh** [isaac-sim/IsaacLab#120] [General]
Hello,

how can I simulate Anymal D robots in Orbit?

Thanks

**Comment by Mayankm96** [comment]
Can you be more specific about what is desired?

In general for integrating any new robot:

1. Import the URDF into USD
2. Modify the robot configuration for that asset. In this case, it should be similar to `ANYMAL_C_CFG`. 

For ANYmal-D, there is no publicly available actuator network. But if you are part of [ANYmal Research](https://www.anymal-research.org/), you may be able to get it through that.

**Comment by Mayankm96** [comment]
We have added support for ANYmal-D. Please note that the actuator network used for the robot corresponds to ANYdrive 3.0 and not ANYdrive 3.1. This may effect sim-to-real transfer for the robot.

If you have an ANYmal Research account, you should be able to obtain the actuator network from there.

**Comment by guiskibh** [comment]
Hello @Mayankm96 , many thanks for your support!
How can I simulate the zoom camera equipped on the payload of the Anymal D? I am looking at this https://isaac-orbit.github.io/orbit/source/tutorials/04_sensors/add_sensors_on_robot.html but I am not sure how it can apply also to a zoom camera.
